src/components/dataset/query/sidebar/panels/InfoView.jsx

```diff
@@ -62,9 +95,43 @@ export class InfoView extends React.PureComponent {
   };
 
   render() {
-    const { classes, dataset } = this.props;
+    const { classes, dataset, user } = this.props;
     const { isAccordionExpanded, isHelpVisible, helpTitle, helpContent } = this.state;
 
+    const linkToBq = dataset.accessInformation?.bigQuery !== undefined;
+    const tables = dataset.schema.tables.map((table, i) => (
+      <li key={`${i}`} className={classes.listItem}>
```

Comment:
Can we make the table list items have borders, or more contrast with the background?

Reply:
sure.  Keep in mind these are gonna go away I think once we have the schema view